src/pyhf/__init__.py

```diff
@@ -101,7 +104,7 @@ def set_backend(backend, custom_optimizer=None):
 
     optimizer_changed = bool(optimizer != new_optimizer)
     # set new backend
-    tensorlib = backend
+    tensorlib = backend(mode=mode.lower())
```

Comment:
`backend` isn't a callable. This isn't valid code.

Reply:
How can I set the mode variable in the backend class? Use `tensorlib.mode = mode.lower()`?